doc/source/projections.rst

```diff
@@ -98,6 +98,41 @@ Re-projecting is the process of changing the representation of locations from on
     ax.set_title("Mercator");
 
 
+Projection for multiple geometry columns
+----------------------------------------
+
+GeoPandas 0.8 implements the support for different projections assigned to different geometry
+columns of the same GeoDataFrame. Projection is now stored together with geometries directly
+on the GeometryArray level.
```

Comment:
```suggestion
columns of the same GeoDataFrame. The projection is now stored together with geometries per column (directly
on the GeometryArray level).
```

(I want to add the "per column", as I think most users will not be familiar with the GeometryArray term)